go/chat/convloader.go

```diff
@@ -426,6 +430,7 @@ func (b *BackgroundConvLoader) load(ictx context.Context, task clTask, uid grego
 		b.Unlock()
 	}()
 
+	job := task.job
```

Comment:
Why did you move this?

Reply:
just leftover from the old diff, will revert